### ISSUE TITLE
Send email doesn't autofill recipient email address

### ISSUE THREAD

**Issue by Spinneruk** [InvoicePlane/InvoicePlane#1000]
## Expected Behavior

Clicking send mail should fill in the clients email address in the To field at the top

## Current Behavior

After clicking send email the To field is empty

## Screenshots

Nothing to show as it's empty 😀

## Steps to Reproduce

1.Find client
2.Create invoice
3.Press send email
4.Next screen has the issue (empty top field)

## Context (Environment)

trying to test the send email to clients, it works if I type the address in manually but doesn't autofill

**Comment by Spinneruk** [comment]
Been doing some testing:

Windows 11 Laptop Microsoft Edge = Works Fine
iMac with Sanoma OS Safari = Doesn't Work
Same iMac with Chrome = Works Fine
Same iMac with Firefox = works when it wants to.

Only one I can get to work all the time is Chrome on Mac and Edge on Windows

Hope that helps (doubt it)

**Comment by nielsdrost7** [comment]
Thanks for the extra information man!
I love the `works when it wants to`, that's a keeper :-)

**Comment by SyfuxX** [comment]
I took a look on that, the bug is basically just because it's a **multiple** input field, which is somehow buggy on safari.

Probably need to build in a javascript fallback solution.

**Comment by Spinneruk** [comment]
Follow up with this.  Ive recently installed the beta for Mac V15.0 Sequoia, when using safari the email to field is filled in 99% of the time...think the 1% is when I've not put in the address correctly by accident.

**Comment by nielsdrost7** [comment]
> think the 1% is when I've not put in the address correctly by accident

@Spinneruk does that mean that we don't need a bugfix?
I've created a branch just in case, but i can't test it locally with Mac.
There are some tools to test it online with some kind of Mac, but I don't know if i can't my local install to show up there.

**Comment by nielsdrost7** [comment]
> think the 1% is when I've not put in the address correctly by accident.

Then can you clarify that one?

And this one?:
> 4.Next screen has the issue (empty top field)

Feel free to throw in some screenshots.
And I still love your humor

**Comment by Spinneruk** [comment]
Yep the 1% was me being a muppet and not typing the address in correctly so ignore that.

I'm unable to throw in the screenshots as it now works on mine 😂, fixed in the latest OS for Mac.  Didn't work in the last one though which was Sonoma.
